backend/controllers/clientController.js

```diff
@@ -0,0 +1,59 @@
+const clientModel = require('../models/Client');
+
+exports.getAllClients = (req, res) => {
+  try {
+    const clients = clientModel.getAllClients();
```

Comment:
Did you mean to do this ?
```suggestion
    const clients = clientModel.find();
```